inc/class-comment-popularity.php

```diff
@@ -204,6 +212,25 @@ public static function activate() {
 		}
 
 		self::set_permissions();
+
+		// Get actual plugin version.
+		$current_version = self::HMN_CP_PLUGIN_VERSION;
+
+		// Get latest version stored in DB option.
+		$plugin_version_db = get_option( 'hmn_cp_plugin_version' );
+		if ( ! $plugin_version_db ) {
+			add_option( 'hmn_cp_plugin_version', $current_version );
+		}
+
+		// vote count field
+		if(!$wpdb->get_var( "SHOW COLUMNS FROM {$wpdb->comments} LIKE 'vote_count'" )) {
```

Comment:
I'm not sure it's a good idea to modify the columns of an existing table, if there are many many rows this will time out and potentially either just fail or create db issues. It's safer to use comment meta for something like this.